tldw_Server_API/app/core/AuthNZ/repos/orgs_teams_repo.py

```
@@ -0,0 +1,1177 @@
+                    removed = row is not None
+                else:
+                    cur = await conn.execute(
+                        """
+                        DELETE FROM team_members
+                        WHERE team_id = ? AND user_id = ?
+                        """,
+                        (team_id, user_id),
+                    )
+                try:
+                    await conn.commit()
+                except Exception as exc:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)